modules/govuk/lib/puppet/parser/functions/data_sync_times.rb

```diff
@@ -0,0 +1,38 @@
+module Puppet::Parser::Functions
+  newfunction(:data_sync_times, :type => :rvalue, :doc => <<-EOS
+Return information about the data sync times, depending on the argument passed
+    EOS
+  ) do |args|
+    start_hour = 22
+    start_minute = 0
+    finish_hour = 8
+    finish_minute = 0
+
+    if args.size != 1
+      raise(ArgumentError, "data_sync_times: Wrong number of arguments " +
+      "(given #{args.size})")
+    end
+
+    type = args[0]
+
+    case type
+    when 'TIME_RANGE'
```

Comment:
Is there a convention for using screaming snake case here? If so I wonder if it's a bit misleading as it looks like it's a constant but it's not. I'd guess in Ruby world a symbol would be most appropriate but I guess that's not available in puppet, though we do use lowercase strings e.g. `"absent"`

Reply:
I can see how it could misleadingly look like a constant. I've now switched to lowercase.